code/modules/vehicles/speedbike.dm

```diff
@@ -0,0 +1,62 @@
+/obj/vehicle/space/speedbike
+	name = "Speedbike"
+	icon = 'icons/obj/bike.dmi'
+	icon_state = "speedbike_blue"
+	layer = OBJ_LAYER
+	keytype = null
+	vehicle_move_delay = 0
+	var/overlay_state = "cover_blue"
+	var/image/overlay = null
+
+/obj/vehicle/space/speedbike/New()
+	..()
+	overlay = image("icons/obj/bike.dmi", overlay_state)
+	overlay.layer = MOB_LAYER + 0.1
+	overlays += overlay
+
+/obj/effect/particle_effect/speedbike_trail
```

Comment:
I'd suggest using `/obj/effect/overlay/temp` for this, as otherwise it copies most of the functionality of `/obj/effect/overlay/temp`